experiment_data.csv

```diff
@@ -0,0 +1 @@
+EXPERIMENT_RELIABILITY,CPU_TYPE,CPU_TEMP,TOTAL_RUNNING_TIME,QUIT_BY_MAX_RUNTIME,QUIT_BY_INDICATOR,QUIT_BY_BACKFILL,MEM_MAX_RSS,OS,INDICATOR_VALUE,BACKFILL_TIMES,PYPERF_AVG_EXPERIMENT_ROUNDS,PYPERF_LAST_TWO_EXPERIMENT_ROUNDS_RATIO,NAME_OF_EXPERIMENT,PYTHON_VERSION
```

Comment:
Is this supposed to be merged into the master too? I'm fine with it but I just want to make sure.

Reply:
Yes, since this is the header of the csv file.